### PR TITLE
UpdateREADME.md and add ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by victoria34**
Issue #110
screencapture-github-victoria34-matchminer-curate-tree-doc-2018-06-29-18_30_47

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,181 +1,43 @@
...
+# MatchMinerCurate
+MatchMinerCurate is a realtime trial curation platform by using [Clinical Trial Markup Language] (CTML) to label 
+patient-specific genomic and clinical data, and structured eligibility criteria for clinical trials. We use Firebase 
+realtime database to store all trials.
+
+## Install Project
+    npm install
+
+## Prepare Firebase Realtime Database
+1. Go to [Firebase Console] to create a project.
```
Comment (JiaoJiao123) [inline README.md]: attache the firebase console link here

File: README.md [hunk 2]
```diff
@@ -1,181 +1,43 @@
...
+    ```
+    cp src/main/webapp/app/environments/environment.example.ts src/main/webapp/app/environments/environment.ts
+    ```
+2. Go to `Authentication` page in Firebase console, click `Web setup` button and Firebase config will pop up. Copy 
+required fields to `environments.ts`.
+    ```html
+    <script src="https://www.gstatic.com/firebasejs/5.1.0/firebase.js"></script>
+    <script>
+      // Initialize Firebase
+      var config = {
```
Comment (JiaoJiao123) [inline README.md]: only from line 28-line 33 are needed. It is confusing to put all content in the Web Set up here

File: README.md [hunk 3]
```diff
@@ -1,181 +1,43 @@
...
+        apiKey: "********",
+        authDomain: "example.firebaseapp.com",
```
Comment (JiaoJiao123) [inline README.md]: leave the value for each key empty string(""), by customizing it like this, people might think they need to change the data somehow.

**Comment by zhx828** [comment]
You need a section describing how to build war file.

File: README.md (hunk 1 above)
Comment (JiaoJiao123) [inline README.md]: 1. Go to [Firebase Console] and login with your Google account.
2. Create a new project by clicking "Add Project".

**Comment by zhx828** [comment]
Let's remove `oncokb: true` from environment.example.ts and set it to false by default.

File: README.md [hunk 4]
```diff
@@ -1,181 +1,43 @@
...
+MatchMinerCurate is a realtime trial curation platform by using [Clinical Trial Markup Language] (CTML) to label 
+patient-specific genomic and clinical data, and structured eligibility criteria for clinical trials. We use Firebase 
+realtime database to store all trials.
+
+## Install Project
+    npm install
+
+## Prepare Firebase Realtime Database
+1. Go to [Firebase Console] to create a project.
+2. Select `Realtime Database` to store and synchronize data in realtime.
```
Comment (JiaoJiao123) [inline README.md]: If you have to add this item, specify where to find it in detail. 
Click "Database" under "Develop" on the left panel and choose `Realtime Databse` .....

But I don't think this is necessary for them to start the project

File: README.md [hunk 5]
```diff
@@ -1,181 +1,43 @@
...
+3. Choose sign-in method and add domain. Go to `Authentication` page and click `Sign-in method`, enable Google as
```
Comment (JiaoJiao123) [inline README.md]: Click "Authentication" under "Develop" section on the left panel and choose `Sign-in method` tab

**Comment by zhx828** [comment]
It should not be called `production: false`, maybe something called `frontEndOnly: true`?

**Comment by zhx828** [comment]
You also need a section describing how to assign the permission. In most scenarios, it's going to be a private database.

**Comment by zhx828** [comment]
Section needed to describe the auto-backup feature.

File: README.md [hunk 6]
```diff
@@ -1,5 +1,71 @@
-# matchminerCurate
-This application was generated using JHipster 4.13.3, you can find documentation and help at [http://www.jhipster.tech/documentation-archive/v4.13.3](http://www.jhipster.tech/documentation-archive/v4.13.3).
+# MatchMinerCurate
+MatchMinerCurate is a realtime trial curation platform by using [Clinical Trial Markup Language] (CTML) to label
```
Comment (zhx828) [inline README.md]: Can we add a link out to the markup language?

File: src/main/webapp/app/service/trial.service.ts [hunk 7]
```diff
@@ -13,8 +13,8 @@ import { environment } from '../environments/environment';
 import { EmailService } from './email.service';
 @Injectable()
 export class TrialService {
-    production = environment.production ? environment.production : false;
-    oncokb = environment.oncokb ? environment.oncokb : false;
+    oncokb = false;
```
Comment (zhx828) [inline src/main/webapp/app/service/trial.service.ts]: We should still allow changing the oncokb through the environment, just don't mention it in the example file.

File: src/main/webapp/app/genomic/genomic.component.ts [hunk 8]
```diff
@@ -31,7 +31,7 @@ export class GenomicComponent implements OnInit {
     'protein_altering', 'splice site_mutation', 'stop_retained', 'synonymous', '3\'UTR', '3_prime_UTR',
     '5\'Flank', '5\'UTR', '5\'UTR_mutation', '5_prime_UTR'];
     annotated_variants = this.trialService.getOncokbVariants();
-    oncokb = environment.oncokb ? environment.oncokb : false;
```
Comment (zhx828) [inline src/main/webapp/app/genomic/genomic.component.ts]: Same here, we need to support setting from environment

Comment (victoria34) [reply src/main/webapp/app/genomic/genomic.component.ts]: This oncokb will fetch value from trial.service.ts, so we just need to maintain only one config variable.

File: README.md (hunk 6 above)
Comment (victoria34) [reply README.md]: I already add the link on line 201.

File: src/main/webapp/app/service/trial.service.ts (hunk 7 above)
Comment (victoria34) [reply src/main/webapp/app/service/trial.service.ts]: I'll roll it back.

File: README.md [hunk 9]
```diff
@@ -1,8 +1,10 @@
...
-
-## Development
+# MatchMinerCurate
+MatchMinerCurate is a realtime trial curation platform by using [Clinical Trial Markup Language] (CTML) to label 
+patient-specific genomic and clinical data, and structured eligibility criteria for clinical trials. We use Firebase 
+realtime database to store all trials.
 
+## Preparation
+### Install Project
 Before you can build this project, you must install and configure the following dependencies on your machine:
```
Comment (zhx828) [inline README.md]: `It is necessary to install and configure the following dependencies:`

**Pull request merged**